src/main_menu.cpp

```diff
@@ -12,15 +12,15 @@ static Cursor cursor = {0,0};
 bool mm_visible;
 
 Line lines[LINES] = {
-    {"cheats", CHEAT_INDEX, "Turn cheats on/off", false},
-    {"flags", FLAGS_INDEX, "Turn in-game flags on/off", false},
-    {"inventory", INVENTORY_INDEX, "Set Link's items and equipment", false},
-    {"memory", MEMORY_INDEX, "Add memory watches to the screen", false},
-    {"practice", PRACTICE_INDEX, "Load practice files", false},
-    {"scene", SCENE_INDEX, "Adjust the scene's state", false},
-    {"settings", SETTINGS_INDEX, "Configure settings", false},
-    {"tools", TOOLS_INDEX, "Use various tools for practice and testing", false},
-    {"warping", WARPING_INDEX, "Warp to dungeons, towns, grottos, etc.", false}};
+    {"cheats", CHEAT_INDEX, "turn cheats on/off", false},
```

Comment:
Is there any reason to make changes to this file? The format of the item descriptions were updated in a previous commit. The only other change is reordering lines 68 and 69, so maybe changes were pushed for this file by mistake.